src/OSPSuite.UI/Views/Charts/ChartEditorView.cs

```diff
@@ -180,5 +181,14 @@ public void SetChartExportSettingsView(IView view)
       {
          panelChartExportSettings.FillWith(view);
       }
+
+      public override void InitializeResources()
+      {
+         base.InitializeResources();
+         layoutColorGrouping.Text = Captions.CurvesColorGrouping;
```

Comment:
I know the other texts were hard coded. But new constants should be moved to the caption file and not hard coded again. 
Please, no hard coded constant in the designer file, ever